arcade/gl/texture.py

```diff
@@ -689,17 +689,25 @@ def write(self, data: Union[bytes, Buffer, array], level: int = 0, viewport=None
             )
 
     def build_mipmaps(self, base: int = 0, max_level: int = 1000) -> None:
-        """Generate mipmaps for this texture. Leaveing the default arguments
-        will usually does the job. Building mipmaps will create several
-        smaller versions of the texture (256 x 256, 128 x 128, 64 x 64, 32 x 32 etc)
-        helping OpenGL in rendering a nicer version of texture
-        when it's rendered to the screen in smaller version.
+        """Generate mipmaps for this texture.
 
-        Note that mipmaps will only be used if the texture filter is
-        configured with a mipmap-type minification::
+        The default values usually work well.
 
-            # Set up linear interpolating minification filter
-            texture.filter = ctx.LINEAR_MIPMAP_LINEAR, ctx.LINEAR
+        Mipmaps are successively smaller versions of an original
+        texture with special filtering applied. Using mipmaps allows
+        OpenGL to render scaled versions of original textures faster
+        and with far fewer scaling artifacts.
```

Comment:
Some of the sources I was reading indicated that it can be used to provide speed increases, but that may have referred to other past implementations. I'll include this change in a bit. :+1: